src/ape_ethereum/ecosystem.py

```diff
@@ -67,6 +67,7 @@
     "goerli": (5, 5),
     "sepolia": (11155111, 11155111),
 }
+BLUEPRINT_HEADER = HexBytes("0xfe7100")
```

Comment:
`0xfe71` is the header, `0x00` after that is the version byte, which is 0 to start, but is combined with a 2 bit value for the size of the length data section